src/main/java/se/sundsvall/customer/api/DetailsResource.java

```diff
@@ -49,11 +45,9 @@ public DetailsResource(final CustomerService customerService) {
     @ApiResponse(responseCode = "500", description = "Internal Server Error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
     @ApiResponse(responseCode = "502", description = "Bad Gateway", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
     public ResponseEntity<CustomerDetailsResponse> getCustomerDetails(@Valid @RequestBody final CustomerDetailsRequest request) {
-        // Make sure that exactly one of partyId or customerEngagementOrgId is provided and non-empty, but not both
-        var partyIdsEmpty = isEmpty(request.getPartyId());
-        var customerEngagementOrgIdIsEmpty = isBlank(request.getCustomerEngagementOrgId());
-        if ((partyIdsEmpty && customerEngagementOrgIdIsEmpty) || (!partyIdsEmpty && !customerEngagementOrgIdIsEmpty)) {
-            throw Problem.valueOf(Status.BAD_REQUEST, "Exactly one of 'partyId' or 'customerEngagementOrgId' must be provided");
+        // Make sure that partyId or customerEngagementOrgId is provided and non-empty
```

Comment:
Extract this to an own Validation annotation (with corresponding validator-class) to keep same pattern as we have for other validations?